package.json

```diff
@@ -11,6 +11,10 @@
   },
   "license": "MPL-2.0",
   "dependencies": {
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
Not sure we need `babel-preset-es2015`.

The `babel-preset-env` should be able to choose the correct presets.

Reply:
This still needs to be here since `babel-preset-env` will select the correct presets, but it still needs the correct presets to be installed. In our case, `es2015` gets selected for iOS 10.

Reply:
AFAIUI, `babel-preset-env` constructs the preset by matching babel's feature flags (in the form of `babel-plugin-transform-es201*-*`) to data from caniuse.com (via `browserlist`).

`babel-preset-es2015` is a hardcoded arrangement of those transforms, i.e. anything `babel-preset-es2015` can do, `browser-preset-env` can do.